JenkinsConsoleUtility/JenkinsScripts/testInit.sh

```diff
@@ -27,6 +27,7 @@ ResetRepo (){
     git checkout master || git checkout -b master || CleanCurrentRepo
     git pull origin master
 
+    # Delete $GitDestBranch, reset it to master, prep for next build and fresh write
     if [ "$GitDestBranch"!="master" ]; then
         git fetch --progress origin
```

Comment:
>git fetch --progress origin [](start = 8, length = 27)

This line doesn't seem to be necessary, we just fetched all the latest stuff above with the same command